homeassistant/components/qbittorrent/sensor.py

```diff
@@ -46,35 +46,35 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
-    """Set up the qBittorrent sensors."""
-
-    try:
-        client = create_client(
-            config[CONF_URL], config[CONF_USERNAME], config[CONF_PASSWORD]
-        )
-    except LoginRequired:
-        _LOGGER.error("Invalid authentication")
-        return
-    except RequestException as err:
-        _LOGGER.error("Connection failed")
-        raise PlatformNotReady from err
-
-    name = config.get(CONF_NAME)
-
-    variables = SENSOR_TYPES
-    sensors = [
-        QBittorrentSensor(
-            sensor_name,
-            client,
-            name,
-            LoginRequired,
-            None,
-        )
-        for sensor_name in variables
-    ]
-
-    add_entities(sensors, True)
+# def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
The Powers That Be will forbid a checkin with commented out code.

Reply:
Was still a work in progress :-)